src/types/formatted/benefitsHub.ts

```diff
@@ -1,13 +0,0 @@
-export type BenefitsHubLink = {
```

Comment:
Removed all `BenefitsHubLink` things as it became a common component (`RelatedLink(s)`)